testadapter/src/execution/TestExecutor.cs

```diff
@@ -80,34 +80,47 @@ public void Run(IFrameworkHandle frameworkHandle, IRunContext runContext, IEnume
             WorkingDirectory = @$"{workingDirectory}"
         };
 
-        using (pProcess = new() { StartInfo = processStartInfo })
+        if (runContext.IsBeingDebugged && frameworkHandle is IFrameworkHandle2 fh2 && frameworkHandle.GetType().ToString().Contains("JetBrains"))
         {
-            pProcess.EnableRaisingEvents = true;
-            pProcess.OutputDataReceived += TestEventProcessor(frameworkHandle, testCases);
-            pProcess.ErrorDataReceived += StdErrorProcessor(frameworkHandle);
-            pProcess.Exited += ExitHandler(frameworkHandle);
-            pProcess.Start();
-            pProcess.BeginErrorReadLine();
-            pProcess.BeginOutputReadLine();
-            AttachDebuggerIfNeed(runContext, frameworkHandle, pProcess);
+            var processId = fh2.LaunchProcessWithDebuggerAttached(processStartInfo.FileName, processStartInfo.WorkingDirectory, processStartInfo.Arguments, null);
```

Comment:
So the new rider version implements the `LaunchProcessWithDebuggerAttached` now?
Can I check the installed `Rider` version?
Otherwise, i need a combination of using  `LaunchProcessWithDebuggerAttached`  and try to catch to use the auto await debugger is attached

Reply:
> So the new rider version implements the LaunchProcessWithDebuggerAttached now?
yes

> Can I check the installed Rider version?
![image](https://github.com/MikeSchulze/gdUnit4Net/assets/1482681/948d7e1f-422e-4fc7-8fab-3db4e9baa9ff)
I have force pushed a check for version of the assembly.

Reply:
Thanks, I will take over your suggestions to the actual PR https://github.com/MikeSchulze/gdUnit4Net/pull/125

Thanks for your help 👍